### PR TITLE
make easy testing

### PR DESCRIPTION
**Pull request by walf443**
this patch make testing by just make command.

### CODE REVIEW

**Comment by walf443** [comment]
if you merge circleci branch, you can do CI on CircleCI!! SEE https://circleci.com/gh/walf443/ngx_dynamic_upstream/2

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+nginx-build.ini:
+	echo "[ngx_dynamic_upstream]\nform=local\nurl=`pwd`" > nginx-build.ini
+
+cpanm:
+	curl -L https://cpanmin.us/ -o cpanm
+	chmod +x cpanm
+
+install-perl-lib: cpanm
+	./cpanm -l tmp/perl Test::More
```
Comment (cubicdaiya) [inline Makefile]: It seems to be not required `Test::More`?

**Comment by walf443** [comment]
fixed deps. prove command is not need to installed.

**Comment by cubicdaiya** [comment]
Thank you for your contribution.

**Pull request merged**